test/integration/rhel-8.0/new-function.patch

```diff
@@ -15,8 +15,8 @@ index 3ad4602..f3cda7e 100644
  	return (b - buf) ? b - buf : retval;
```

Comment:
Note for future work: we should probably explain this in the patch author guide.

Reply:
> Note for future work: we should probably explain this in the patch author guide.

I'm working on it now, I'll post up a PR with author guide changes soon.